### DISCUSSION TITLE
Does Camoufox support Socks5 proxies?

### DISCUSSION THREAD

**Discussion by Basit2121** [daijro/camoufox#77] [Q&A]
trying to start a browser instance with a socks5 proxy, if i dont use the geoip = True i get this warning 
\main.py:112: LeakWarning: When using a proxy, it is heavily recommended that you pass `geoip=True`.
  with Camoufox(proxy={"server": "socks5://127.0.0.1:9050"}, locale="en-US", screen=constrains, humanize=True, block_images=True
  
  and if i do use geoip = True then i get this error 
  
  Proxy.__init__() missing 2 required positional arguments: 'username' and 'password'
  
  CODE :
  
  constrains = Screen(max_width=1280, max_height=960)
            with Camoufox(geoip=True ,proxy={"server": "socks5://127.0.0.1:9050"}, locale="en-US", screen=constrains, humanize=True, block_images=True
            ) as browser:

**Comment by daijro** [comment]
Fixed by #84 :+1:
